examples/timeline/other/drag&drop.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE HTML>
+<html>
+<head>
```

Comment:
Sorry I just noticed that the UTF-8 coding is missing: `<meta charset="utf-8"/>`
Without it the unicode-symbols are broken:
![image](https://cloud.githubusercontent.com/assets/600565/19817264/26d04fc0-9d4b-11e6-8bd9-d235ecec7cc8.png)